TracyOpenCL.hpp

```diff
@@ -21,6 +21,7 @@
 #define TracyCLZoneSetEvent(e)
 
 #define TracyCLCollect(c)
+#define TracyCLBlockingCollect(c)
```

Comment:
Unfortunately I don't think this should be provided as an option. It should be the user's responsibility to ensure that all events are finished before a call to `TracyCLCollect`. I understand your reasoning but the profiler should not block.

If the user launches an async operation it shouldn't be the profiler responsibility to wait for it.